iceoryx_hoofs/legacy/include/iceoryx_hoofs/internal/cxx/storable_function.hpp

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2023 by Apex.AI Inc. All rights reserved.
```

Comment:
@FerdinandSpitzschnueffler I think you can delete this file here. As this is internal API, there is no need to provide a legacy header.